util/custom-response/src/main/java/com/brother/customresponse/payload/ErrorResponseDto.java

```diff
@@ -0,0 +1,14 @@
+package com.brother.customresponse.payload;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
```

Comment:
Please use only required annotations.
Example: @Getter, @Setter and etc.

Reply:
> Please use only required annotations. Example: @getter, @Setter and etc.

Done